skimage/restoration/tests/test_denoise.py

```diff
@@ -386,6 +386,49 @@ def test_wavelet_denoising():
         assert_(np.sum(res1**2) <= np.sum(res2**2))
 
 
+def test_wavelet_denoising_scaling():
+    """Test case for floating point input outside the [-1, 1] range."""
+    rstate = np.random.RandomState(1234)
+    for dtype in [np.float16, np.float32, np.float64, np.int16, np.uint8]:
```

Comment:
Possibly this should be done with `pytest.parametrize`, which will then tell you which types failed, if any.